docs/explanation/upgrade-groups.rst

```diff
@@ -0,0 +1,64 @@
+==============
+Upgrade Groups
+==============
+
+In the process of planning or executing an upgrade for an OpenStack cloud, users have the
+capability to target a specific group of components based on their operational roles. 
+
+Control Plane
+-------------
+
+The **control-plane** includes services tasked with making decisions related to data management,
+routing, and processing. Services considered as **control-plane** in OpenStack and in scope
+of COU are:
+
+- rabbitmq-server
+- ceph-mon
+- keystone
+- aodh
+- barbican
+- ceilometer
+- ceph-fs
+- ceph-radosgw
+- cinder
+- designate
+- designate-bind
+- glance
+- gnocchi
+- heat
+- manila
+- manila-ganesha
+- neutron-api
+- neutron-gateway
+- ovn-dedicated-chassis
+- ovn-central
+- placement
+- nova-cloud-controller
+- openstack-dashboard
+- octavia
+- subordinate services that are not collocated with **data-plane** nodes 
+
+
+Data Plane
+----------
+
+On the other hand, the **data-plane** is composed of services that handle the actual data
+transfer. Services considered as **data-plane** in OpenStack and in scope of COU are:
+
+- nova-compute
+- ceph-osd
+- any **control-plane** services collocated on the same nodes with data-plane components
```

Comment:
My suggestion is just to reflect the current implementation, but if this sentence is true we might have to reconsider. @aieri is there any chance of a hyper converged deployment that ceph-osd is with a `control-plane` app but not with nova-compute? Or this would be considered as a bad deployment?

E.g: ceph-osd and glance in a machine

```suggestion
- any **control-plane** services colocated on the same machine with nova-compute application
```